src/grafana/chart/templates/dashboards.yaml

```diff
@@ -9,380 +9,9 @@ metadata:
-                        "placement": "bottom",
-                        "showLegend": true
-                    },
-                    "tooltip": {
-                        "hideZeros": false,
-                        "mode": "single",
-                        "sort": "none"
-                    }
-                },
-                "pluginVersion": "11.6.1",
-                "targets": [
-                    {
-                        "expr": "sum(container_memory_usage_bytes{namespace=\"$namespace\", container!=\"\", pod!~\".*waypoint.*\"})",
-                        "legendFormat": "With Sidecar",
-                        "refId": "A"
-                    },
-                    {
-                        "expr": "sum(container_memory_usage_bytes{namespace=\"$namespace\", container!=\"\", container!=\"istio-proxy\"})",
-                        "legendFormat": "Without Sidecar",
-                        "refId": "B"
-                    }
-                ],
-                "title": "Memory Usage - All Workloads (With vs Without Istio Sidecar)",
-                "type": "timeseries"
-            }
-        ],
-        "preload": false,
-        "refresh": "",
-        "schemaVersion": 41,
-        "tags": [
-            "istio",
-            "uds"
-        ],
-        "templating": {
-            "list": [
-                {
-                    "datasource": "Prometheus",
-                    "definition": "label_values(kube_pod_info, namespace)",
-                    "includeAll": false,
-                    "label": "Namespace",
-                    "name": "namespace",
-                    "query": {
-                        "query": "label_values(kube_pod_info{namespace!~\"kube-system|istio-system|istio-admin-gateway|istio-tenant-gateway|istio-passthrough-gateway|pepr-system|neuvector|loki|monitoring|grafana|vector|velero|keycloak|authservice|zarf|uds-dev-stack\"}, namespace)",
-                        "refId": "StandardVariableQuery"
-                    },
-                    "type": "query"
-                }
-            ]
-        },
-        "time": {
-            "from": "now-1h",
-            "to": "now"
-        },
-        "timepicker": {
-            "refresh_intervals": [
-                "5s",
-                "10s",
-                "30s",
-                "1m",
-                "5m",
-                "15m",
-                "30m",
-                "1h"
-            ]
-        },
-        "timezone": "",
-        "title": "Istio Sidecar vs Ambient Resource Comparison",
-        "description": "A summary of CPU and Memory usage by namespace. Provides estimated resource savings when switching from Sidecar to Ambient mode.",
-        "uid": "cemp0hgd41ssge",
-        "version": 1
-    }
+  # Load dashboards from files
+  {{- $files := .Files }}
+  {{- range $path, $bytes := .Files.Glob "dashboards/*.json" }}
+  {{ base $path }}: |-
+    {{ $files.Get $path | nindent 4 }}
+  {{- end }}
```

Comment:
I might lean towards putting the new Keycloak dashboards into the Keycloak chart, similar to what we do with [Loki](https://github.com/defenseunicorns/uds-core/blob/1623-introduce-keycloak-grafana-dashboards/src/loki/chart/templates/loki-dashboards.yaml). 

I can see arguments for both approaches though, and it may matter less here since Grafana requires Keycloak so there's no situation where you'd have the dashboard without Keycloak itself.